### DISCUSSION TITLE
keyball61_左側トラックボールなし_12Pコンスルーへの換装につきまして

### DISCUSSION THREAD

**Discussion by JinBallMen** [Yowkees/keyball#329] [Q&A]
以前、組み立てる際に以下の画像のように
OLEDの右側についてジャンパに失敗しました。パッド剥離したため、エナメル線で繋いで運用しています。

キーボード自体は正常に使用できていますが、ピンソケットよりコンスルーのほうが背が低く、またProMicroの換装もしやすいと考えた為、ピンソケットを外してコンスルーにしたいと考えています。

そうした時に、以下の画像のような状態のため今までエナメル線で繋げていた箇所に、今まで通り線で繋げるとコンスルーにした意味がなくなってしまうと考えています。

もし、代わりに繋げられる箇所がありましたら、ご教示いただけますでしょうか。

![20230905_233710~3](https://github.com/Yowkees/keyball/assets/75989686/e42d10f0-bddb-4bfc-8df0-cbc79916ce90)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

それでは、緑点で示したスルーホールは使用しませんので、こちらにつないでください。
黒枠内の上から５番目の穴です。

よろしくお願いいたします。
![KB61_OLEDfix2](https://github.com/Yowkees/keyball/assets/84209772/adb16676-8981-464d-9d4e-742e12ec0aa9)
